rules/subrepo_rules.build_defs

```diff
@@ -206,13 +210,17 @@ def github_repo(name:str, repo:str, revision:str, build_file:str=None, hashes:st
 
     return new_http_archive(
         name = name,
-        urls = [f'https://github.com/{org}/{repo}/archive/{revision}.zip'],
+        urls = [
+            f'https://github.com/{org}/{repo}/archive/{revision}.zip',
+            f'https://api.github.com/repos/{org}/{repo}/zipball/{revision}'
```

Comment:
So the rule tries to strip the first level of the directory structure so that instead of `$OUT/pleasings-1.0.0/.plzconfig` we get just the repo in the root i.e. `$OUT/.plzconfig`. 

The problem is that when you download using the `https://api.github.com` approach, we get a different root directory, `thought-machine-pleasings-b85bbd0`, where the last part is the commit has for the `1.0.0` tag. This isn't determinable from the parameters so we can't set `strip_prefix` correctly. I don't know if there's a nice solution to this...

Reply:
thanks! I'll look at this tonight and look for workarounds. I've used --shell quite a bit. I think it's my lack of familiarity with the bootstrap.sh process. I was just running the bootstrap command and did not know how to use plz inside the container I setup to compile/test plz.

Reply:
You can just `plz plz build //some:remote_file` and that will build please from source, and run `plz build //some:remote_file`.

Reply:
So I was able to make this trick work. I had to run bootstrap first and then I could do it.